### ISSUE TITLE
Azimuth and Signac error 

### ISSUE THREAD

**Issue by nitinmahajan20** [satijalab/azimuth#202]
I use Sagemaker and Posit workbench and have Seurat 5.0.1 

Having issues running the following codes 
```{r}
remotes::install_github("satijalab/azimuth", "seurat5", quiet = TRUE)
remotes::install_github("stuart-lab/signac", "seurat5", quiet = TRUE)
```
Error I am getting is 
```
Error: Failed to install 'Azimuth' from GitHub: HTTP error 404. No commit found for the ref seurat5  Did you spell the repo owner (`stuart-lab`) and repo name (`signac`) correctly? - If spelling is correct, check that you have the required permissions to access the repo.
```

I did try with the "quiet=FALSE" as well, even then  its is not working 

Looked at this as well "https://github.com/satijalab/azimuth/issues/187" 

But not working !!!
```
> sessionInfo()
R version 4.2.3 (2023-03-15)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 18.04.6 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/openblas/libblas.so.3
LAPACK: /usr/lib/x86_64-linux-gnu/libopenblasp-r0.2.20.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8    LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C             LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] BiocManager_1.30.22 Seurat_5.0.1        SeuratObject_5.0.1  sp_2.1-2           

loaded via a namespace (and not attached):
  [1] Rtsne_0.17             colorspace_2.1-0       deldir_2.0-2           ellipsis_0.3.2        
  [5] ggridges_0.5.5         fs_1.6.3               RcppHNSW_0.5.0         spatstat.data_3.0-4   
  [9] rstudioapi_0.15.0      leiden_0.4.3.1         listenv_0.9.0          remotes_2.4.2.1       
 [13] ggrepel_0.9.5          RSpectra_0.16-1        fansi_1.0.6            codetools_0.2-19      
 [17] splines_4.2.3          cachem_1.0.8           knitr_1.45             pkgload_1.3.4         
 [21] polyclip_1.10-6        spam_2.10-0            jsonlite_1.8.8         ica_1.0-3             
 [25] cluster_2.1.4          png_0.1-8              uwot_0.1.16            spatstat.sparse_3.0-3 
 [29] shiny_1.8.0            sctransform_0.4.1      compiler_4.2.3         httr_1.4.7            
 [33] Matrix_1.6-4           fastmap_1.1.1          lazyeval_0.2.2         cli_3.6.2             
 [37] later_1.3.2            htmltools_0.5.7        tools_4.2.3            igraph_1.6.0          
 [41] dotCall64_1.1-1        gtable_0.3.4           glue_1.7.0             RANN_2.6.1            
 [45] reshape2_1.4.4         dplyr_1.1.4            Rcpp_1.0.12            scattermore_1.2       
 [49] jquerylib_0.1.4        vctrs_0.6.5            nlme_3.1-162           spatstat.explore_3.2-5
 [53] progressr_0.14.0       lmtest_0.9-40          spatstat.random_3.2-2  stringr_1.5.1         
 [57] xfun_0.41              globals_0.16.2         ps_1.7.6               mime_0.12             
 [61] miniUI_0.1.1.1         lifecycle_1.0.4        irlba_2.3.5.1          devtools_2.4.5        
 [65] goftest_1.2-3          future_1.33.1          MASS_7.3-58.2          zoo_1.8-12            
 [69] scales_1.3.0           spatstat.utils_3.0-4   promises_1.2.1         parallel_4.2.3        
 [73] RColorBrewer_1.1-3     yaml_2.3.8             curl_5.2.0             memoise_2.0.1         
 [77] gridExtra_2.3          reticulate_1.34.0      pbapply_1.7-2          ggplot2_3.4.4         
 [81] sass_0.4.8             stringi_1.8.3          desc_1.4.3             fastDummies_1.7.3     
 [85] pkgbuild_1.4.3         rlang_1.1.3            pkgconfig_2.0.3        matrixStats_1.2.0     
 [89] evaluate_0.23          lattice_0.20-45        tensor_1.5             ROCR_1.0-11           
 [93] purrr_1.0.2            patchwork_1.2.0.9000   htmlwidgets_1.6.4      cowplot_1.1.2         
 [97] processx_3.8.3         tidyselect_1.2.0       parallelly_1.36.0      RcppAnnoy_0.0.21      
[101] plyr_1.8.9             magrittr_2.0.3         R6_2.5.1               profvis_0.3.8         
[105] generics_0.1.3         pillar_1.9.0           fitdistrplus_1.1-11    abind_1.4-5           
[109] survival_3.5-3         tibble_3.2.1           future.apply_1.11.1    KernSmooth_2.23-20    
[113] utf8_1.2.4             spatstat.geom_3.2-7    plotly_4.10.4          urlchecker_1.0.1      
[117] rmarkdown_2.25         usethis_2.2.2          grid_4.2.3             data.table_1.14.10    
[121] callr_3.7.3            digest_0.6.34          xtable_1.8-4           tidyr_1.3.0           
[125] httpuv_1.6.13          munsell_0.5.0          viridisLite_0.4.2      bslib_0.6.1           
[129] sessioninfo_1.2.2  

```

**Comment by dchauss** [comment]
I am having the exact same issue using R trying to make a new build.

**Comment by Gesmira** [comment]
Hi, 
The "seurat5" branch for signac no longer exists and all of the applicable updates have been merged into the `master` branch. The `master` branch of Azimuth is also up to date. Thus, you should just install with the default settings for both packages. 

```
remotes::install_github('satijalab/azimuth', ref = 'master')
remotes::install_github('stuart-lab/signac', ref = 'master') # or you can install directly from CRAN
```

**Comment by dchauss** [comment]
Thanks so much for the clarification! Much appreciated.Best,Daniel Sent from my iPhoneOn Jan 25, 2024, at 9:16 AM, gesmira ***@***.***> wrote:﻿
Closed #202 as completed.
